src/en/news/blog/2023/v18-2-0-reef-released/index.md

```diff
@@ -0,0 +1,921 @@
+  accordingly. The following is a list of the field names impacted by this
+  change:
+
+  - ``absolute``
+
+  - ``add``
+
+  - ``admin``
+
+  - ``appendable``
+
+  - ``bucket_key_enabled``
+
+  - ``delete_marker``
+
+  - ``exists``
+
+  - ``has_bucket_info``
+
+  - ``high_precision_time``
+
+  - ``index``
+
+  - ``is_master``
+
+  - ``is_prefix``
+
+  - ``is_truncated``
+
+  - ``linked``
+
+  - ``log_meta``
+
+  - ``log_op``
+
+  - ``pending_removal``
+
+  - ``read_only``
+
+  - ``retain_head_object``
+
+  - ``rule_exist``
+
+  - ``start_with_full_sync``
+
+  - ``sync_from_all``
+
+  - ``syncstopped``
+
+  - ``system``
+
+  - ``truncated``
+
+  - ``user_stats_sync``
+
+- The Beast front end's HTTP access log line now uses a new
+  ``debug_rgw_access`` configurable. It has the same defaults as
+  ``debug_rgw``, but it can be controlled independently.
+
+- The pubsub functionality for storing bucket notifications inside Ceph
+  has been removed. As a result, the pubsub zone should not be used anymore.
+  The following have also been removed: the REST operations, ``radosgw-admin``
+  commands for manipulating subscriptions, fetching the notifications, and
+  acking the notifications. 
+
+  If the endpoint to which the notifications are sent is down or disconnected,
+  we recommend that you use persistent notifications to guarantee their
+  delivery. If the system that consumes the notifications has to pull them
+  (instead of the notifications being pushed to the system), use an external
+  message bus (for example, RabbitMQ or Kafka) for that purpose. 
+
+- The serialized format of notification and topics has changed. This means
+  that new and updated topics will be unreadable by old RGWs. We recommend
+  completing the RGW upgrades before creating or modifying any notification
+  topics.
+
+- Compression is now supported for objects uploaded with Server-Side
+  Encryption. When both compression and encryption are enabled, compression is
+  applied before encryption. Earlier releases of multisite do not replicate
+  such objects correctly, so all zones must upgrade to Reef before enabling the
+  `compress-encrypted` zonegroup feature: see
+  https://docs.ceph.com/en/reef/radosgw/multisite/#zone-features and note the
+  security considerations.
+
+### Telemetry
+
+- Users who have opted in to telemetry can also opt in to
+  participate in a leaderboard in the telemetry public dashboards
+  (https://telemetry-public.ceph.com/). In addition, users are now able to
+  provide a description of their cluster that will appear publicly in the
+  leaderboard. For more details, see:
+  https://docs.ceph.com/en/reef/mgr/telemetry/#leaderboard. To see a sample
+  report, run ``ceph telemetry preview``. To opt in to telemetry, run ``ceph
+  telemetry on``. To opt in to the leaderboard, run ``ceph config set mgr
+  mgr/telemetry/leaderboard true``. To add a leaderboard description, run
+  ``ceph config set mgr mgr/telemetry/leaderboard_description ‘Cluster
```

Comment:
Are the quoting marks in _‘Cluster description’_ correct? Tiny-little-minor-nit.